addons/token-exchange/blue_secret_controller.go

```diff
@@ -100,16 +108,23 @@ func (c *blueSecretTokenExchangeAgentController) sync(ctx context.Context, syncC
 		return nil
 	}
 
+	err = c.enableCSIOmapGenerator()
+	if err != nil {
+		return fmt.Errorf("failed to start omap-generator for rook: %v", err)
+	}
```

Comment:
move this to MirrorPeer controller. Let this controller focus on secret exchange only.